sklearn/metrics/tests/test_common.py

```diff
@@ -1736,6 +1737,11 @@ def test_metrics_pos_label_error_str(metric, y_pred_threshold, dtype_y_str):
 def check_array_api_metric(
     metric, array_namespace, device, dtype_name, y_true_np, y_pred_np, sample_weight
 ):
+    if _array_api_skips.get(metric.__name__, {}).get(array_namespace) == "all":
+        pytest.skip(
+            f"{array_namespace} is not Array API compliant for {metric.__name__}"
```

Comment:
```suggestion
            f"Skipping {metric.__name__} because of missing Array API compliance in {array_namespace}"
```
It feels like the current text is backwards. What do you think of this change? Trying to make it clear that "we are skipping testing for X because something in {array_namespace}'s Array API support is missing" - which is what I think we are doing.